source/sonata_simulation.rst

```diff
@@ -96,6 +96,27 @@ example::
        "forward_skip": 500
   }
 
+LFPWeightsPath
+--------------
+
+*Optional*
+A parameter specifying the path to the weights file describing the scaling factors for the contributions of each segment's transmembrane current to the LFP/EEG. The format of the weights file is as follows:
+
+.. table::
+  =========================== ======================= ========== ============= =============================================================================================================================================================================================================
+  Group                       Field                   Type       Requirement   Description
+  =========================== ======================= ========== ============= =============================================================================================================================================================================================================
+  /electrodes/{electrodename} layer                   str        Optional      Layer of the circuit in which {electrodename} is located
+  /electrodes/{electrodename} location                float64    Optional      Position of {electrodename} in cartesian coordinates (3 x 1)
```

Comment:
you can call it position then. It can be float32 as neuron are already in float32.
Also, please add the unit.